UpdateService/handler.ts

```diff
@@ -155,7 +155,11 @@ const updateServiceTask = (
         body: {
           ...retrievedService,
           ...servicePayload,
-          service_id: serviceId
+          service_id: serviceId,
+          service_metadata: {
+            ...servicePayload.service_metadata,
+            token_name: retrievedService.service_metadata.token_name // TODO check on ADB2C with userInfo data
```

Comment:
can you keep this pr in draft until we fix this one?

Reply:
Done 👍